code/src/index.css

```diff
@@ -1,13 +1,283 @@
+html {
+  background: url(./pink.jpeg);
+  background-position-x: center;
+  background-position-y: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed; 
+  background-size: cover;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  box-sizing: border-box;
+  font-family: 'Open Sans', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.container {
+  height: 100vh;
+  width: 320px; 
+  margin: 0 auto;
+  display: grid;
+  align-items: center;
+}
+
+.question-container {
+  padding: 22px 28px 140px 28px;
+  display: grid;
+  text-align: left;
+  background-color: rgba(230, 230, 230, 0.3);
+  position: relative;
+  height: 40vh;
+}
+
+/* NAME-COMPONENT */
+.name-question {
+  margin-top: 37px;
+}
+
+.name-box {
+ margin-top: 15px;
+} 
+
+input[type=text] {
+  border: none;
+  background-color: rgba(218, 130, 156, 0.5);
+  font-size: 18px;
+}
+
+.positionButton {
+  position: absolute;
+  right: 18px;
+  bottom: 18px;
+}
+
+/* LOCATION-COMPONENT */
+.location-select {
+  background-color: pink;
+  height: fit-content;
+  font-size: 16px;
+}
+
+.locationButtons {
+  position: absolute;
+  right: 18px;
+  bottom: 18px;
+  display: inline-grid;
+  grid-template-columns: 1fr 1fr; 
+  column-gap: 8px;
+}
+
+ /* STYLING RADIOBUTTONS */
+ .radio-toolbar input[type="radio"] {
+  display: none;
+}
+
+.radio-toolbar {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+.radio-toolbar label {
+  display: inline-block;
+  background-color: rgb(157, 92, 92);
+  padding: 7px 16px;
+  font-size: 16px;
+  border-radius: 0;
+  margin: 5px;
+}
+
+.radio-toolbar input[type="radio"]:checked + label {
+  background-color:#bfb;
+}
+
+.radio-toolbar input[type="radio"]:focus + label {
+  border: 2px dashed #444;
+}
+
+.radio-toolbar label:hover {
+  background-color: white;
+}
+
+.radiobuttons {
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-items: center;
+    row-gap: 10px;
+    margin-top: 38px;
+} 
+
+.radiobuttons input[type="radio"] {
+  opacity: 0;
+  position: fixed;
+  width: 0;
+}
+
+
+.radiobuttons label {
+  display: inline-block; 
+  background-color: rgb(157, 92, 92);
+  padding: 5px 10px;
+  font-size: 16px;
+  border-radius: 0;
+  width: 110px;
+  align-self: center;
+  justify-self: start;
+  
+}
+
+.radiobuttons input[type="radio"]:checked + label {
+  background-color:#bfb; 
+
+}
+
+.radiobuttons input[type="radio"]:focus + label {
+  border: 2px dashed #444;
+}
+
+.radiobuttons label:hover {
+  background-color: white;
+}
+
```

Comment:
don't know if this works, but I was thinking that if you want to show which radiobutton has been clicked you could try:
.radiobuttons label:visited